crypto/evp/evp_enc.c

```diff
@@ -269,7 +269,11 @@ int EVP_CipherInit_ex(EVP_CIPHER_CTX *ctx, const EVP_CIPHER *cipher,
         case NID_sm4_ofb128:
         case NID_rc4:
         case NID_rc4_40:
-        break;
+        case NID_rc5_cbc:
+        case NID_rc5_ecb:
+        case NID_rc5_cfb64:
+        case NID_rc5_ofb64:
+            break;
```

Comment:
... someone could remind me why we don't just try an implicit fetch first, and *only if that fails*, tries to use the legacy stuff...

Reply:
Not sure. @mattcaswell might have an answer?